torch/csrc/autograd/init.cpp

```diff
@@ -202,20 +211,22 @@ PyObject* THPAutograd_initExtension(PyObject* _unused, PyObject *unused) {
   py::class_<ProfilerResult>(m, "_ProfilerResult")
     .def("trace_start_us", &ProfilerResult::trace_start_us)
     .def("events", &ProfilerResult::events)
-    .def("save", &ProfilerResult::save);
+#ifdef USE_KINETO
+    .def("save", &ProfilerResult::save)
```

Comment:
Also chrome trace saving is available only if you use KINETO?

Reply:
if kineto is not available we use a python implementation for trace export (in profiler_util.py)

Reply:
Ok thats good.